ArduCopter/Copter.cpp

```diff
@@ -802,7 +802,7 @@ void Copter::one_hz_loop()
     if (!using_rate_thread) {
         attitude_control->set_notch_sample_rate(AP::scheduler().get_filtered_loop_rate_hz());
     }
-    pos_control->get_accel_z_pid().set_notch_sample_rate(AP::scheduler().get_filtered_loop_rate_hz());
```

Comment:
I wonder if we could change "_u_" to "_up_"?

Reply:
I think we resolved on `U`